examples/semantic_segmentation/README.md

```diff
@@ -54,16 +51,14 @@ Run a script using the `--help` flag for more information:
 
 ```shell
 $ poetry run python3 semantic_segmentation/seed_test_run.py --help
-usage: seed_test_run.py [-h] [--model MODEL]
-                        [--test_suites TEST_SUITES [TEST_SUITES ...]]
+usage: seed_test_run.py [-h] [--model MODEL] [--test_suites TEST_SUITES [TEST_SUITES ...]]
                         --out_bucket OUT_BUCKET
 
-optional arguments:
+options:
   -h, --help            show this help message and exit
   --model MODEL         Name of model in directory to test
   --test_suites TEST_SUITES [TEST_SUITES ...]
                         Name(s) of test suite(s) to test.
   --out_bucket OUT_BUCKET
-                        Name of AWS S3 bucket with write access to
-                        upload result masks to.
+                        Name of AWS S3 bucket with write access to upload result masks to.
```

Comment:
Is this the actual output from `poetry run python3 semantic_segmentation/seed_test_run.py --help`?

Reply:
Ya, I copied it over